tests/_support/AcceptanceTester.php

```diff
@@ -235,4 +235,19 @@ public function doAdministratorLogout()
 		$I->waitForText('Log in', 60, ['xpath' => "//fieldset[@class='loginform']//button"]);
 
 	}
+
+	/**
+	 * Selects an option in a Joomla Radio Field based on its label
+	 *
+	 * @return void
+	 */
+	public function selectOptionInRadioField($label, $option)
+	{
+		$I = $this;
+		$I->comment("Trying to select the $option from the $label");
+		$label = $webDriver->findField(['xpath' => "//label[contains(normalize-space(string(.)), '" . $label . "')]"]);
```

Comment:
I'll investigate more, but probably we will have to create a helper to deal with joomla Chosen JS.  Have a look at http://codeception.com/docs/06-ModulesAndHelpers. 

We can also check the Locator Util http://codeception.com/docs/reference/Locator maybe we can make it work using it.